OrcScala/src/orc/run/StandardOrcRuntime.scala

```diff
@@ -19,8 +19,9 @@ import orc.run.extensions._
 
 class StandardOrcRuntime(engineInstanceName: String) extends Orc(engineInstanceName)
   with StandardInvocationBehavior
-  //with OrcWithWorkStealingScheduler
-  with OrcWithThreadPoolScheduler
+  with OrcWithWorkStealingScheduler
```

Comment:
The only way to know is to test it. I'm not sure it's stable enough for release. But it's stable enough for master.